simulations/vip-408/ethereum.ts

```diff
@@ -0,0 +1,345 @@
+          });
+
+          it(`should set ${symbol} collateral factor to ${params.collateralFactor}`, async () => {
+            const market = await comptroller.markets(vTokens[symbol]);
+            expect(market.collateralFactorMantissa).to.equal(parseUnits(params.collateralFactor, 18));
+          });
+
+          it(`should set ${symbol} liquidation threshold to ${params.liquidationThreshold}`, async () => {
+            const market = await comptroller.markets(vTokens[symbol]);
+            expect(market.liquidationThresholdMantissa).to.equal(parseUnits(params.liquidationThreshold, 18));
+          });
+
+          it(`should set ${symbol} protocol seize share ${params.protocolSeizeShareMantissa}`, async () => {
+            expect(await vToken.protocolSeizeShareMantissa()).to.equal(
+              parseUnits(params.protocolSeizeShareMantissa, 18),
+            );
+          });
+
+          it(`should set ${symbol} supply cap to ${params.supplyCap}`, async () => {
+            expect(await comptroller.supplyCaps(vTokens[symbol])).to.equal(
+              parseUnits(params.supplyCap, underlyingDecimals),
+            );
+          });
+
+          it(`should set ${symbol} borrow cap to ${params.borrowCap}`, async () => {
+            expect(await comptroller.borrowCaps(vTokens[symbol])).to.equal(
+              parseUnits(params.borrowCap, underlyingDecimals),
+            );
+          });
+        });
+      }
+    });
+
+    describe("Pools configuration", () => {
+      const checkComptroller = (comptrollerAddress: string, comptrollerName: string) => {
+        describe(`${comptrollerName} Comptroller`, () => {
+          let comptroller: Contract;
+
+          before(async () => {
+            comptroller = await ethers.getContractAt(COMPTROLLER_ABI, comptrollerAddress);
+          });
+
+          it("should use the correct comptroller address", async () => {
+            expect(comptroller.address).to.equal(comptrollerAddress);
+          });
+
+          it("should have the correct price oracle", async () => {
+            expect(await comptroller.oracle()).to.equal(ethereum.RESILIENT_ORACLE);
+          });
+
+          it("should have close factor = 0.5", async () => {
+            expect(await comptroller.closeFactorMantissa()).to.equal(parseUnits("0.5", 18));
+          });
+
+          it("should have liquidation incentive = 1.04", async () => {
+            expect(await comptroller.liquidationIncentiveMantissa()).to.equal(parseUnits("1.04", 18));
+          });
+
+          it("should have minLiquidatableCollateral = $100", async () => {
+            expect(await comptroller.minLiquidatableCollateral()).to.equal(parseUnits("100", 18));
+          });
+
+          it("should have owner = GUARDIAN", async () => {
+            expect(await comptroller.owner()).to.equal(ethereum.NORMAL_TIMELOCK);
+          });
+        });
+      };
+
+      checkComptroller(COMPTROLLER_ETHENA, "Ethena");
```

Comment:
Could we also call `checkIsolatedPoolsComptrollers`?

Reply:
[1866d37](https://github.com/VenusProtocol/vips/pull/440/commits/1866d377e77845c38ab5502842a9fee5915bfbb9)